{"name": "Search book with multiple result", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x00007FF6B5DD4D02+56194]\n\t(No symbol) [0x00007FF6B5D404B2]\n\t(No symbol) [0x00007FF6B5BE76AA]\n\t(No symbol) [0x00007FF6B5C316D0]\n\t(No symbol) [0x00007FF6B5C317EC]\n\t(No symbol) [0x00007FF6B5C74D77]\n\t(No symbol) [0x00007FF6B5C55EBF]\n\t(No symbol) [0x00007FF6B5C72786]\n\t(No symbol) [0x00007FF6B5C55C23]\n\t(No symbol) [0x00007FF6B5C24A45]\n\t(No symbol) [0x00007FF6B5C25AD4]\n\tGetHandleVerifier [0x00007FF6B614D5BB+3695675]\n\tGetHandleVerifier [0x00007FF6B61A6197+4059159]\n\tGetHandleVerifier [0x00007FF6B619DF63+4025827]\n\tGetHandleVerifier [0x00007FF6B5E6F029+687785]\n\t(No symbol) [0x00007FF6B5D4B508]\n\t(No symbol) [0x00007FF6B5D47564]\n\t(No symbol) [0x00007FF6B5D476E9]\n\t(No symbol) [0x00007FF6B5D38094]\n\tBaseThreadInitThunk [0x00007FF875657344+20]\n\tRtlUserThreadStart [0x00007FF8769426B1+33]", "trace": "self = <testcases.ui.scenario_2.TestScenario1 object at 0x0000023B51BF17F0>\naccount = <data_object.account.Account object at 0x0000023B51C20A70>\nbook = <data_object.book.Book object at 0x0000023B51C20B00>\n\n    @allure.title(\"Search book with multiple result\")\n    @pytest.mark.parametrize(\"account\", Account.get_list_account_from_json('resources/test_data/scenario_2.json','valid'))\n    @pytest.mark.parametrize(\"book\", Book.get_list_book_from_json('resources/test_data/book_scenario_2.json','valid'))\n    def test_search_book_with_ui(self, account, book):\n        Report.report_step('Go to Log in page')\n        book_store_page =  BookStorePage()\n        # book_store_page.go_to_login_page()\n    \n        # Report.report_step('Log in to the application')\n        # login_page = LoginPage()\n        # login_page.login(account)\n    \n        # Report.report_step('using search function')\n        # book_store_page.wait_for_page_load()\n        book_store_page.using_search_book_function(book.book_name)\n>       book_store_page.verify_book(book.book_name)\n\ntestcases\\ui\\scenario_2.py:32: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npage_object\\book_store_page.py:39: in verify_book\n    a =self.book_locator.get_attribute('id')\ncore\\element\\element.py:56: in get_attribute\n    self.wait_for_visibility()\ncore\\element\\element.py:17: in wait_for_visibility\n    self._element = WebDriverWait(self._driver, timeout).until(EC.visibility_of_element_located((self._by[0] , self._by[1])))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"da90458da816e751514461ea1dae09db\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x0000023B51C2D4E0>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       \tGetHandleVerifier [0x00007FF6B5DD4D02+56194]\nE       \t(No symbol) [0x00007FF6B5D404B2]\nE       \t(No symbol) [0x00007FF6B5BE76AA]\nE       \t(No symbol) [0x00007FF6B5C316D0]\nE       \t(No symbol) [0x00007FF6B5C317EC]\nE       \t(No symbol) [0x00007FF6B5C74D77]\nE       \t(No symbol) [0x00007FF6B5C55EBF]\nE       \t(No symbol) [0x00007FF6B5C72786]\nE       \t(No symbol) [0x00007FF6B5C55C23]\nE       \t(No symbol) [0x00007FF6B5C24A45]\nE       \t(No symbol) [0x00007FF6B5C25AD4]\nE       \tGetHandleVerifier [0x00007FF6B614D5BB+3695675]\nE       \tGetHandleVerifier [0x00007FF6B61A6197+4059159]\nE       \tGetHandleVerifier [0x00007FF6B619DF63+4025827]\nE       \tGetHandleVerifier [0x00007FF6B5E6F029+687785]\nE       \t(No symbol) [0x00007FF6B5D4B508]\nE       \t(No symbol) [0x00007FF6B5D47564]\nE       \t(No symbol) [0x00007FF6B5D476E9]\nE       \t(No symbol) [0x00007FF6B5D38094]\nE       \tBaseThreadInitThunk [0x00007FF875657344+20]\nE       \tRtlUserThreadStart [0x00007FF8769426B1+33]\n\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:105: TimeoutException"}, "steps": [{"name": "Go to Log in page", "status": "passed", "start": 1702564339441, "stop": 1702564339441}], "attachments": [{"source": "06a77828-1053-417b-b8b5-45ef13ffcccd-attachment.png", "type": "image/png"}, {"name": "stdout", "source": "833c2052-9032-4975-b39b-24f6d54ee4a5-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "book", "value": "<data_object.book.Book object at 0x0000023B51C20B00>"}, {"name": "account", "value": "<data_object.account.Account object at 0x0000023B51C20A70>"}], "start": 1702564339441, "stop": 1702564344846, "uuid": "f3d326af-bd31-480b-9cf6-bbdc8aba2004", "historyId": "42d9edf458bfd8e6484ed40a598c57d6", "testCaseId": "044c20361cb403e4ec16e1960bbd95b1", "fullName": "testcases.ui.scenario_2#test_search_book_with_ui", "labels": [{"name": "parentSuite", "value": "UI Test"}, {"name": "suite", "value": "Test Scenario 2"}, {"name": "subSuite", "value": "TestScenario1"}, {"name": "host", "value": "VNNOT02222"}, {"name": "thread", "value": "1948-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.ui.scenario_2"}]}